application/src/main/java/org/thingsboard/server/controller/BaseController.java

```diff
@@ -376,9 +377,18 @@ private ThingsboardException handleException(Exception exception, boolean logExc
             return new ThingsboardException("Unable to send mail: " + exception.getMessage(), ThingsboardErrorCode.GENERAL);
         } else if (exception instanceof AsyncRequestTimeoutException) {
             return new ThingsboardException("Request timeout", ThingsboardErrorCode.GENERAL);
-        } else {
-            return new ThingsboardException(exception.getMessage(), exception, ThingsboardErrorCode.GENERAL);
+        } else if (exception instanceof DataAccessException) {
```

Comment:
:+1:

Reply:
We can't publish SQL errors to user's browser. Please log it with the corresponding log level but hide it from the user.